### PR TITLE
General cleanup for maps and robot.java as well as removing lift and control panel

### PR DESCRIPTION
**Pull request by andrewmartin5**
Killed off lift and control panel

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -64,7 +63,6 @@
     final private Shooter shooter = new Shooter(map.getShooterMap());
     // final private ControlPanel controlPanel = new
     // ControlPanel(map.getControlPanelMap());
```
Comment (bot190) [inline src/main/java/frc/robot/Robot.java]: This should be deleted.

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -88,12 +86,11 @@ public void robotInit() {
         SmartDashboard.putData("unloadBallToShooter", indexer.unloadBall());
         SmartDashboard.putData("ShootNoSpinup", indexer.shootBall());
         SmartDashboard.putData("index ball", indexer.indexBall());
-        SmartDashboard.putData("lift brake toggle", lift.toggleBrake());
         SmartDashboard.putData("Deploy intake", intake.deployIntake());
         SmartDashboard.putData("Retract intake", intake.retractIntake());
 
         SmartDashboard.putData("cam toggle", camToggle());
-        SmartDashboard.putData("After Match Lift Sequence", afterMatchPit());
+        SmartDashboard.putData("After Match Retract Intake", intake.retractIntake());
```
Comment (bot190) [inline src/main/java/frc/robot/Robot.java]: Do we need this? I'm pretty sure it defaults to retracting.

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -106,7 +103,6 @@ public void robotInit() {
         Shuffleboard.getTab("Shuffleboard").add("Autonomous", autoChooser);
 
         drive.setDefaultCommand(drive.drive(driveController::getTriggers, () -> driveController.getX(Hand.kLeft)));
-        lift.setDefaultCommand(lift.moveLift(() -> -copilotController.getTriggers()));
         // controlPanel.setDefaultCommand(controlPanel.spinControlPanel(() ->
```
Comment (bot190) [inline src/main/java/frc/robot/Robot.java]: This and other references to control panel should be removed.

File: src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -224,14 +214,14 @@ public CommandBase systemsCheck() {
 
     // in the future we will add the vision lining up command to this.
     public CommandBase endGame() {
-        final CommandBase endGameCmd = new SequentialCommandGroup(intake.deployIntake(), lift.disengageRatchet());
+        final CommandBase endGameCmd = new SequentialCommandGroup(intake.deployIntake());
```
Comment (bot190) [inline src/main/java/frc/robot/Robot.java]: This was only useful with the lift. No need to deploy the intake during endgame now.

**Review by bot190 — COMMENTED** [review]
There's some more control panel code to remove

File: src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (andrewmartin5) [reply src/main/java/frc/robot/Robot.java]: actually yeah it does

File: src/main/java/frc/robot/Robot.java (hunk 4 above)
Comment (andrewmartin5) [reply src/main/java/frc/robot/Robot.java]: so can we just kill that entire command?

Comment (msoucy) [reply src/main/java/frc/robot/Robot.java]: Why was it a group of one thing‽

Comment (andrewmartin5) [reply src/main/java/frc/robot/Robot.java]: ¯\\_(ツ)_/¯

File: src/main/java/frc/robot/maps/FrancoisMap.java [hunk 5]
```diff
@@ -49,7 +49,7 @@ public SmartSpeedController getRight() {
 
                 PIDSparkMax sendLeader = new PIDSparkMax(rightLeader);
                 sendLeader.getEncoder().setPositionScaleFactor(distancePerRev);
-                sendLeader.getEncoder().setVelocityScaleFactor(distancePerRev);
+                sendLeader.getEncoder().setVelocityScaleFactor(distancePerRev / 60);
```
Comment (msoucy) [inline src/main/java/frc/robot/maps/FrancoisMap.java:54]: Comment why this is done

File: src/main/java/frc/robot/Robot.java [hunk 6]
```diff
@@ -287,15 +265,11 @@ private void configureButtonBindings() {
         driveController.getButton(Button.kBumperRight).whenHeld(drive.slowTurn(true));
         driveController.getButton(Button.kBumperLeft).whenHeld(drive.slowTurn(false));
 
-        // copilotController.getButton(Button.kB).whenPressed(controlPanel.stageTwoRotation());
-        // copilotController.getButton(Button.kX).whenPressed(controlPanel.stageThreeRotation());
         copilotController.getButton(Button.kA).whenHeld(intake.intake());
         copilotController.getButton(Button.kBumperRight).whenPressed(shooter.spinUp(4400));
         copilotController.getButton(Button.kBumperLeft).whenHeld(shooter.spinDown());
         final XboxTrigger endTrigger = new XboxTrigger(copilotController, Hand.kRight);
```
Comment (msoucy) [inline src/main/java/frc/robot/Robot.java]: Is this trigger used anywhere?

**Review by msoucy — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 6 above)
Comment (andrewmartin5) [reply src/main/java/frc/robot/Robot.java]: not really; should i just murder it?

**Review by bot190 — APPROVED** [review]

**Review by msoucy — APPROVED** [review]

**Pull request merged**